web/babel.config.json

```diff
@@ -1,6 +1,11 @@
 {
   "presets": [
-    "@babel/preset-env",
+    [
+      "@babel/preset-env",
+      {
+        "useBuiltIns": "entry"
```

Comment:
We define this here, and it mostly just means "support most browsers, but not so many that it makes modern features break"